apps/passport-server/migrations/39_multiple_anon_topics.sql

```diff
@@ -0,0 +1,14 @@
+-- Create a new table to map telegram_chat_id to multiple anon_topic_id rows
+create table telegram_chat_anon_topics (
+	ticket_event_id UUID NOT NULL, 
+	anon_topic_id BIGINT NOT NULL, 
+	anon_topic_name VARCHAR(255) NOT NULL,
+	primary key (ticket_event_id, anon_topic_id),
```

Comment:
Why is it indexed by ticket_event_id and not telegram_chat_id? Multiple ticket_event_ids can point to the same Telegram group